manifests/d/Dman95/SASM/3.14.4/Dman95.SASM.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Dman95.SASM
+PackageVersion: 3.14.4
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+ReleaseDate: 2023-03-03
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/Dman95/SASM/releases/download/v3.14.0/SASMSetup3140.exe

```

Comment:
This is the previous version `InstallerUrl`. Apparently [the PR's version](https://github.com/Dman95/SASM/releases/download/v3.14.4/SASMSetup3144.exe) isn't available yet:
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/7b5aec16-b318-4412-b3d0-789c5ca9e48f)